client/python/gradio_client/client.py

```diff
@@ -1557,7 +1557,8 @@ def __next__(self) -> tuple | Any:
                     o = self.communicator.job.outputs[self._counter]
                     self._counter += 1
                     return o
-                if self.communicator.job.latest_status.code == Status.FINISHED:
+                if self.communicator.job.latest_status.code == Status.FINISHED and \
```

Comment:
```suggestion
                if (
                    self.communicator.job.latest_status.code == Status.FINISHED
                    and self._counter >= len(self.communicator.job.outputs)
                ):
```

Need this to pass the lint

Reply:
Did that.  I ignored what that script did to some other random file, and only committed the file I changed.